### DISCUSSION TITLE
PAT Basic 1058. 选择题 (C语言实现) (20) | PAT Solution in C

### DISCUSSION THREAD

**Discussion by xlucn** [xlucn/PAT#55] [General]
https://oliverlew.github.io/PAT/Basic/1058.html 

 题目

**Comment by xyycz** [comment]
请问这个题的测试点3的测试条件是什么啊？

**Comment by xlucn** [comment]
> @1216569960
> 请问这个题的测试点3的测试条件是什么啊？

PAT的测试点是不公开的，只能自己找代码漏洞

**Comment by xyycz** [comment]
> @OliverLew
> > @1216569960
> > 请问这个题的测试点3的测试条件是什么啊？
> 
> PAT的测试点是不公开的，只能自己找代码漏洞

需要考虑答案输入的顺序比如bac和score求和超范围么

**Comment by xlucn** [comment]
> @1216569960
>
> 需要考虑答案输入的顺序比如bac

这就要审题了：“注意每题的选项从小写英文字母 a 开始顺次排列。”

> 和score求和超范围么

一共最多就100道题，每题不超过5分，简单分析就知道不会。

总之，99%的题都是很严谨的，每一句话都要留意，条件都给你了。

**Comment by xyycz** [comment]
> @OliverLew
> > @1216569960
> >
> > 需要考虑答案输入的顺序比如bac
> 
> 这就要审题了：“注意每题的选项从小写英文字母 a 开始顺次排列。”
> 
> > 和score求和超范围么
> 
> 一共最多就100道题，每题不超过5分，简单分析就知道不会。
> 
> 总之，99%的题都是很严谨的，每一句话都要留意，条件都给你了。

好的，谢谢您
